aten/src/ATen/cuda/CUDAEvent.h

```diff
@@ -13,6 +13,16 @@
 #include <cstdint>
 #include <utility>
 
+/*
+* `cudaEventTimingOnly` is a torch-specific flag that is used to indicate that
+* the CUDAEvent will only be used for timing, and never for synchronization.
+* CUDAEvents used for intra-graph timing must be recorded with `cudaEventRecordExternal`,
+* whereas CUDAEvents used for inter-graph synchronization must never be recorded with
+* `cudaEventRecordExternal`; `cudaEventTimingOnly` enables the distinction between these
+* two use cases. `cudaEventEnableTiming` must be set in conjunction with `cudaEventTimingOnly`.
+*/
+#define cudaEventTimingOnly 0x05
```

Comment:
I don't think this is correct, flags are bits, and 1, 2, 4 are already taken, so your next option is 8, not 5, and even that is not future proof - what if nvidia decides to add some other flags that will conflict?

Reply:
Ah yes you're totally right, changing to 0x08. I'm not sure how we could be 100% future proof here, but it should be simple enough to update if nvidia does add new flags. Also I'm hoping we eventually consolidate the API back to the original form, through some form of deprecation cycle, and so this won't be around forever.